src/segmentation_a_direct.jl

```diff
@@ -112,7 +112,9 @@ Compute the gradient magnitude of an image using the Sobel operator.
 """
 function imgradientmag(img)
     h = centered([-1 0 1; -2 0 2; -1 0 1]')
-    Gx = imfilter(img, h', "replicate")
-    Gy = imfilter(img, h, "replicate")
+    Gx_future = Threads.@spawn IceFloeTracker.imfilter(img, h', "replicate")
```

Comment:
Why does Gx_future appluy to `h'`, but Gy_future applies to `h`? Just wanted to double check that it's not a typo.

Reply:
That is how the algo works in MATLAB -- see the code in the `imgradientxy.m` function.